### PR TITLE
added hangman

### PR DESCRIPTION
**Pull request by zakomon**

### CODE REVIEW

File: [Hangman]_[Rahul_Narayanan] [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 0f5d6160457d6cf1093a6006849c915a644c6020
```
Comment (mananwason) [inline [Hangman]_[Rahul_Narayanan]:1]: @zakomon Please add the actual files not the commit numbers. Thanks!

**Comment by mariobehling** [comment]
"This branch has conflicts that must be resolved". Please solve it before tomorrow, so we can accept your task in GCI. Thank you!

**Comment by jigyasa-grover** [comment]
@SaptakS I believe let the student respond, before we close this PR ...

**Comment by SaptakS** [comment]
@jig08 the student has already made the same PR which was merged.. So, I thought it would be ok to close this one..

**Comment by jigyasa-grover** [comment]
Oh, Mention the new PR number here then ..

**Comment by SaptakS** [comment]
#29

**Comment by jigyasa-grover** [comment]
Awesome.

**Pull request closed without merging**